Main.py

```diff
@@ -0,0 +1,94 @@
+import pygame, sys, time, math, random
+from pygame.locals import *
+
+pygame.init()#Initiates Pygame
```

Comment:
Inline comments should be [tabbed across](https://www.python.org/dev/peps/pep-0008/#inline-comments) according to PEP-8 guidelines.